app-frontend/package.json

```diff
@@ -1,146 +1,147 @@
 {
-  "name": "raster-foundry",

```

Comment:
Please ignore the whitespace here to view the actual changes. I believe that Prettier did this.